EPNTAP.tex

```diff
@@ -764,6 +764,9 @@ \subsubsection{Target description}
 associate dwarf\_planet\#asteroid, so as to remove any ambiguity
 (e.g., to help support older catalogues).
 
+\item If several target\_name values are provided for a granule, 
+one-to-one correspondance with target\_class is expected when possible.
```

Comment:
The second part of the sentence is not so much useful as it doesn't say how to make this one-to-one correspondance. Do we mean same cardinality of elements and same number of elements in the hash-separated-lists ? For instance: 
- target_class = planet#satellite#satellite
- target_name = Jupiter#Io#Europa 

It's probably better just to state that there can be several elements in the target_class. We may refine this in the future.

Reply:
This won't do when a dwarf planet is present - hence "when possible". But OK, it doesn't matter anyway